src/migrations/20210328210805-add-verification-user.js

```diff
@@ -0,0 +1,25 @@
+const { generator } = require('../services/customAlphabet');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Users', 'isVerified', {
+      type: Sequelize.BOOLEAN,
+      allowNull: false,
+      defaultValue: false,
+    });
+
+    await queryInterface.addColumn('Users', 'verificationHash', {
+      type: Sequelize.STRING,
+      defaultValue: generator,
+    });
+
+    return queryInterface.bulkUpdate('Users', {
+      isVerified: true,
+    });
```

Comment:
I noticed that after migrating the column `verificationHash` is all null. So we would have the state `isVerified=false` but `verificationHash=null` 

I couldn't find a good way to populate `verificationHash`, so this is my workaround for now cuz I wanna merge